### ISSUE TITLE
FreeType2 bug: Casting CString::as_ptr() to mut is UB

### ISSUE THREAD

**Issue by qwerty19106** [twistedfall/opencv-rust#534]
Code:
```rust
use opencv::freetype::{create_free_type2, FreeType2Trait};

fn main() {
    println!("000");
    let mut font = create_free_type2().unwrap();
    println!("111");
    font.load_font_data("RubikDoodleShadow-Regular.ttf", 0)
        .unwrap();
    println!("222");
}
```
returns:
```
000
111
thread 'main' panicked at /home/pc/.cargo/registry/src/index.crates.io-6f17d22bba15001f/opencv-0.88.6/src/traits/opencv_type.rs:299:9:
not implemented: Casting CString::as_ptr() to mut is UB
```

1. Raspberry Pi OS (Debian 12)
2. OpenCV: package
3. OpenCV version: 4.6.0
4. rustc 1.77.0-nightly (595bc6f00 2024-01-05)

**Comment by twistedfall** [comment]
Thanks for the report, I'll check it out!

**Comment by twistedfall** [comment]
This issue should now be fixed in v0.88.7
